pom.xml

```diff
@@ -155,6 +155,8 @@
         <netty.version>4.1.72.Final</netty.version>
         <netty-codec-http2-version>4.1.72.Final</netty-codec-http2-version>
         <jersey-common>2.34</jersey-common>
+
+        <maven.surefire.version>3.0.0-M5</maven.surefire.version>
```

Comment:
Any reason not to use 3.0.0-M6? 

Or is an issue caused by that newer version?

Reply:
I'm waiting to see if the jenkins build passes with this pin, the confluent commons got updated to M6 which is when build errors started causing. As to why the new version causes build errors, I haven't dug into that yet.